lib/state/appActions.js

```diff
@@ -257,6 +258,13 @@ appActions.login = function(credentials, options, cb) {
       return cb(err);
     }
     self._logMetric(self.trackedState.LOGIN_SUCCESS);
+    
+    var acks = localStore.getItem('acknowledgments');
```

Comment:
I `devices` see is retrieved in `_afterLoginPage`. I could move this logic into that function as it is over ever called after this point and before when we render the page.